### PR TITLE
Update data-service-get-started.md 

### PR DESCRIPTION
**Pull request by sgykfjsm**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix https://github.com/pingcap/docs/issues/15743

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by heibaijian — APPROVED** [review]

**Review by qiancai — COMMENTED** [review]
Hi @sgykfjsm, I've added some comments to this PR. PTAL. Thanks.

File: tidb-cloud/data-service-get-started.md [hunk 1]
```diff
@@ -131,9 +133,31 @@ You can customize SQL statements for the endpoint in the SQL editor, which is th
...
+    For Windows or Linux:
+
+    - If you have only one query in the editor, press **Ctrl + Enter** or click <svg width="1rem" height="1rem" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6.70001 20.7756C6.01949 20.3926 6.00029 19.5259 6.00034 19.0422L6.00034 12.1205L6 5.33028C6 4.75247 6.00052 3.92317 6.38613 3.44138C6.83044 2.88625 7.62614 2.98501 7.95335 3.05489C8.05144 3.07584 8.14194 3.12086 8.22438 3.17798L19.2865 10.8426C19.2955 10.8489 19.304 10.8549 19.3126 10.8617C19.4069 10.9362 20 11.4314 20 12.1205C20 12.7913 19.438 13.2784 19.3212 13.3725C19.307 13.3839 19.2983 13.3902 19.2831 13.4002C18.8096 13.7133 8.57995 20.4771 8.10002 20.7756C7.60871 21.0812 7.22013 21.0683 6.70001 20.7756Z" fill="currentColor"></path></svg>**Run** to execute it.
+
+    - If you have multiple queries in the editor, select the lines of the target queries with your cursor, and then press **Ctrl + Enter** or click **Run** to execute them sequentially.
+
+    - To run all queries in the editor sequentially, press **Shift + Ctrl + Enter**, or select the lines of all queries with your cursor and click **Run**.
+
+    </div>
+    </SimpleTab>
```
[the remark this answers is not in the document]
Comment (sgykfjsm) [reply tidb-cloud/data-service-get-started.md:161]: Fixed at https://github.com/pingcap/docs/pull/16112/commits/119633bc73845027a16721cc0f5f153279358743

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/data-service-get-started.md [hunk 2]
```diff
@@ -131,9 +133,31 @@ You can customize SQL statements for the endpoint in the SQL editor, which is th
 
     If you have inserted parameters in the SQL statements, make sure that you have set test values or default values for the parameters in the **Params** tab on the right pane. Otherwise, an error is returned.
 
-    To run a SQL statement, select the line of the SQL with your cursor and click **Run** > **Run at cursor**.
+    <SimpleTab>
+    <div label="macOS">
+
```
Comment (qiancai) [inline tidb-cloud/data-service-get-started.md:138]: Tip for reviewers:

Line 136~L160 are the same with that in  https://github.com/pingcap/docs/blob/release-7.5/tidb-cloud/data-service-manage-endpoint.md?plain=1#L158-L182

File: tidb-cloud/data-service-get-started.md [hunk 3]
```diff
@@ -153,7 +177,7 @@ To deploy the endpoint, perform the following steps:
 
 2. Click **Deploy** to confirm the deployment. You will get the **Endpoint has been deployed** prompt if the endpoint is successfully deployed.
 
-    On the right pane of the endpoint details page, you can click the **Deployments** tab to view the deployed history.
+    To view the deployment history, you can click the name of your Data App in the left pane, and then click the **Deployments** tab on the right pane.
```
Comment (qiancai) [inline tidb-cloud/data-service-get-started.md:180]:
```suggestion
    To view the deployment history, you can click the name of your Data App in the left pane, and then click the **Deployments** tab in the right pane.
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**